tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -97,30 +99,51 @@
 # .. role:: green
 # .. role:: red
 #
-# 1. Neuromag head coordinate frame ("head", :pink:`pink axes`)
-#      Defined by the intersection of 1) the line between the LPA
-#      (:red:`red sphere`) and RPA (:purple:`purple sphere`), and
-#      2) the line perpendicular to this LPA-RPA line one that goes through
-#      the Nasion (:green:`green sphere`).
-#      The axes are oriented as **X** origin→RPA, **Y** origin→Nasion,
+# 1. Neuromag/Elekta/MEGIN head coordinate frame ("head", :pink:`pink axes`)
+#      The head coordinate frame is defined through the coordinates of
+#      anatomical landmarks on the subject's head: Usually the Nasion (`NAS`_),
```

Comment:
in brackets, fiducials

Reply:
why? Anatomical landmarks are physical locations on a subject's head ... fiducials are objects that are placed near to or on anatomical landmarks, ... to then identify them in an MRI or other picture.

An example are vitamin E pills, which show pretty well in MRIs: https://pdfs.semanticscholar.org/36d7/61e642000231ca0c66392aa0f3563a3eff81.pdf

Reply:
oh I see. You always learn something new :)